### PR TITLE
Parse ellipsis (...) in the old parser

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

File: grammar/Python.asdl [hunk 1]
```diff
@@ -83,6 +83,8 @@ module LPython
          | ConstantBool(bool value, string? kind)
          | ConstantFloat(float value, string? kind)
          | ConstantComplex(float re, float im, string? kind)
+         | ConstantEllipsis(string? kind)
```
Comment (namannimmo10) [inline grammar/Python.asdl:86]: How are we going to support this in the ASR?

Comment (certik) [reply grammar/Python.asdl:86]: I think `...` is similar to `pass`, so that's just an empty subroutine, so you leave the `body` argument empty in ASR.

Comment (namannimmo10) [reply grammar/Python.asdl:86]: What I've noticed is that people use `Ellipsis` to indicate something they want to fill in later (like a todo empty block) and `pass` stmt to mean a block is intended to have _no_ code. 

`...` is used in NumPy code, too, as a placeholder in slice notation. 
See more uses here: https://python.land/python-ellipsis

Comment (certik) [reply grammar/Python.asdl:86]: It has different uses. For the "empty block" use, we can just represent that with an empty block in ASR.

**Pull request merged**